spec/venue_spec.rb

```diff
@@ -44,12 +44,46 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  # Iteration 3
+
+  describe '#over_capacity' do
+    it 'can tell if it is over capacity' do
+
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      expect(venue.over_capacity?).to be(false)
+      venue.add_patron('James')
```

Comment:
So you can see here in this test that we would only be over_capacity if our `@patrons` array in greater than capacity (see line 61)